pkg/ffuf/job.go

```diff
@@ -367,19 +394,36 @@ func (j *Job) CheckStop() {
 		}
 	}
 
-	// check for maximum running time
+	// Check for runtime of entire process
 	if j.Config.MaxTime > 0 {
 		dur := time.Now().Sub(j.startTime)
 		runningSecs := int(dur / time.Second)
 		if runningSecs >= j.Config.MaxTime {
-			j.Error = "Maximum running time reached, exiting."
+			j.Error = "Maximum running time for entire process reached, exiting."
 			j.Stop()
 		}
 	}
+
+	// Check for runtime of current job
+	if j.Config.MaxTimeJob > 0 {
+		dur := time.Now().Sub(j.startTimeJob)
+		runningSecs := int(dur / time.Second)
+		if runningSecs >= j.Config.MaxTimeJob {
+			j.Output.Info(fmt.Sprintf("Maximum running time for this job reached, continuing with next job if one exists."))
```

Comment:
I think this should not directly output the info, but instead set the `j.Error` like the full process stop case does.

```suggestion
			j.Error = "Maximum running time for this job reached, continuing with next job if one exists."
```

Reply:
Yep, noticed that as well. Fixed.